src/modules/common/popup/popup.component.js

```diff
@@ -20,9 +20,12 @@ export default class PopupComponent extends Component {
      * Hides a popup
      *
      * @method hidePopup
+     * @param {SyntheticEvent} event
      * @memberof PopupComponent
      */
-    hidePopup() {
+    hidePopup(event) {
+        event.stopPropagation();
```

Comment:
What was the reason to stop event propagation?

Reply:
Now there is no exact reason for this, but there was during development.

Popup is rendered within `SelectedContentComponent`. 
So there is that chain of events: 
1. you click on popup.
2. `hidePopup` hides popup, and calls `onClose` from `SelectedContentComponent`
3. click event reaches *top div* of `SelectedContentComponent` 

What is important is that in some point of developement there was `togglePopup` as `onClick` on the *top div* of `SelectedContentComponent`. 
So back then event from point 3 was reaching top div, and in turn `togglePopup` was called so popup was reopened again. Because of `togglePopup` being called.

event reaches popup -> hidePopup -> event reaches *top div* -> togglePopup 
hide + toggle === no change

But at the end I moved `onClick` to another div and there is no longer need for `stopPropagation `.

Remove this `stopPropagation`?

Reply:
Yes, now it's not needed here.